src/EventLogExpert/MainPage.xaml.cs

```diff
@@ -16,8 +16,6 @@
 using Microsoft.Extensions.Logging;
 using Microsoft.Maui.Platform;
 using System.Collections.Immutable;
-using System.Diagnostics;
-using System.Diagnostics.Eventing.Reader;

```

Comment:
Sorry, should have clarified better. We need System.Diagnostics.Eventing.Reader, but didn't need the System.Diagnostics that was added in this PR.